cypress/README.md

```diff
@@ -54,10 +54,11 @@ TODO allow using {id:...} instead of string label for menu items, gtl items, tre
 
 #### GTL
 
-* `cy.gtl()` - return GTL
 * `cy.gtl_error` - check that error message is present
 * `cy.gtl_no_record` - check that `No data` message is present
-* `cy.gtl_click('Name')` - click on a GTL
+* `cy.gtlGetTable` - return GTL table
+* `cy.gtlGetRows(columns)` - return GTL table row data in an array. columns: Array of ints of columns to read. Example: [1,2,3] will return all row data from columns 1,2 and 3.
+* `cy.gtlClickRow(columns)` - click on a row in a GTL table. columns: Array of { title: String, number: String }. Title is the string you want to find in the table to click on, number is the column that string is found in. Example: [{title: 'Default', number: '1'}, {title: 'Compute', number: '2'}] will click on a row in the GTL table with 'Default' in column 1 and 'Compute' in column 2. Using just [{title: 'Default', number: '1'}] will click on the first row found in the GTL table with 'Default' in column 1.
```

Comment:
```suggestion
* `cy.gtlClickRow(columns)` - click on a row in a GTL table. columns: Array of `{ title: String, number: String }`. `title` is the string you want to find in the table to click on, `number` is the column that string is found in. (e.g. `[{title: 'Default', number: '1'}, {title: 'Compute', number: '2'}]` will click on a row in the GTL table with 'Default' in column 1 and 'Compute' in column 2. Using just `[{title: 'Default', number: '1'}]` will click on the first row found in the GTL table with 'Default' in column 1.
```

Reply:
One other comment here - I notice that number is a String.  If you pass an _actual_ number will that not work?

Reply:
Yeah thats a good point. Changed to number and repushed.